luma/core/virtual.py

```diff
@@ -190,10 +190,9 @@ class snapshot(hotspot):
     updates.
     """
     def __init__(self, width, height, draw_fn=None, interval=1.0):
-        assert interval > 0
```

Comment:
This looks to be a small regression that has crept back in, same at line 195/196, please revert this change. Caught by the tests here: https://travis-ci.org/github/rm-hull/luma.core/jobs/716425608#L851

Reply:
@dhrone I mean't that the method should not be modified by your PR, and should be:

``python
    def __init__(self, width, height, draw_fn=None, interval=1.0):
        assert interval > 0
        super(snapshot, self).__init__(width, height, draw_fn)
        self.interval = interval
        self.last_updated = -interval
```

This was a fix implemented in #179

Reply:
Ok.  Sorry for the confusion.  I believe I've reverted this change back to its original state as well as the change to the test_viewport.py test.